notes/interview.vbs

```diff
@@ -3287,14 +3328,14 @@ function display_expedited_dialog()
 				PushButton 270, 257, 195, 13, "Information about EBT Cards", ebt_card_info_btn
 
 			End If
-			GroupBox 5, 295, 470, 60, "If you need support in handling for expedited, please access these resources:"
-			PushButton 15, 305, 150, 13, "HSR Manual - Expedited SNAP", hsr_manual_expedited_snap_btn
-			PushButton 15, 320, 150, 13, "HSR Manual - SNAP Applications", hsr_snap_applications_btn
-			PushButton 15, 335, 150, 13, "SIR - SNAP Expedited Flowchart", sir_exp_flowchart_btn
-			PushButton 165, 305, 150, 13, "Retrain Your Brain - Expedited - Identity", ryb_exp_identity_btn
-			PushButton 165, 320, 150, 13, "Retrain Your Brain - Expedited - Timeliness", ryb_exp_timeliness_btn
-			PushButton 315, 305, 150, 13, "CM 04.04 - SNAP / Expedited Food", cm_04_04_btn
-			PushButton 315, 320, 150, 13, "CM 04.06 - 1st Mont Processing", cm_04_06_btn
+			GroupBox 5, 315, 470, 60, "If you need support in handling for expedited, please access these resources:"
+			PushButton 15, 325, 150, 13, "HSR Manual - Expedited SNAP", hsr_manual_expedited_snap_btn
+			PushButton 15, 340, 150, 13, "HSR Manual - SNAP Applications", hsr_snap_applications_btn
```

Comment:
HSR Manual- SNAP Applications give me a "page not found" error
CM 04.06- 1st Mont Processing spelling error, correct to "Month"

Reply:
Looks like the SNAP Applications page is gone. I created another issue to resolve this #1469 since it will also need to be updated in NOTES - CAF and NOTES - Expedited Determination.

Spelling error fixed!